src/views/modal/registration-js.php

```diff
@@ -76,7 +80,7 @@ class="tribe-tickets__item__attendee__fields__form <?php echo sanitize_html_clas
 			$notice_classes[] = 'tribe-common-a11y-hidden';
 		}
 
-		tribe( 'tickets.editor.template' )->template(
+		$template->template(
```

Comment:
`$this`?

Reply:
nope - this gets used in the modal and the modal doesn't have the template function carry through.